### PR TITLE
[MICROBA-744] Refactor to support themeing the Learner record and certs

### PR DESCRIPTION
**Pull request by staubina**
This PR consists of the following changes

- Moved a large number of files around. 
-- I created new folders in the themes to support theming the base css files `main-ltr.scss`, `main-rtl.scss`, and supporting files separately from the certificate css files. 
-- copied over the existing default styles from Credentials as a starting point
- I updated the weback script to support building both types of files with different names. (This can be improved later, it is a bit rough right now.)
- Added a new base.html file to apply the main styles if it is present.
- Installed `paragon` and `edx-brand` into the package-json.

This resulted in a large number of listed file changes

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ function generateConfig(theme) {
     let config = {
         cache: true,
```
Comment (staubina) [inline webpack.config.js:10]: Note: File I actually changed

File: edx_credentials_themes/templates/edx.org/credentials/base.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
```
Comment (staubina) [inline edx_credentials_themes/templates/edx.org/credentials/base.html:1]: Notes: New file

**Review by Tj-Tracy — APPROVED** [review]
The webpack config LGTM 👍

File: edx_credentials_themes/static/mitpe/certificates/css/mitpe.certificate.style-ltr.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
```
Comment (tuchfarber) [inline edx_credentials_themes/static/mitpe/certificates/css/mitpe.certificate.style-ltr.css:8]: It looks like the new config is placing the files at `.../mitpe/certificates/css/...` and the `base.html` is looking for them at `.../mitpe/css/...`. How much easier would this be if we just delete the MITPE stuff before attempting this? It's a pretty risk-less PR to make and we should just ping George for a :+1: . Then you won't be trying to deal with the "old way" and the "new way" at the same time.

Comment (staubina) [reply edx_credentials_themes/static/mitpe/certificates/css/mitpe.certificate.style-ltr.css:8]: I will look into this now.

**Pull request closed without merging**